### PR TITLE
IRIS-3901 - strip trailing slashes from wikiUrl

### PR DESCRIPTION
**Pull request by katanka**
https://wikia-inc.atlassian.net/browse/IRIS-3901

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [FounderController.class.php#L106](https://github.com/Wikia/app/blob/d7fbdf873a830ec77a2f879f9359b7ed381c8761/extensions/wikia/Email/Controller/FounderController.class.php#L106): Immediately return this expression instead of assigning it to the temporary variable "$snippet". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. ![MINOR][MINOR] [FounderController.class.php#L344](https://github.com/Wikia/app/blob/d7fbdf873a830ec77a2f879f9359b7ed381c8761/extensions/wikia/Email/Controller/FounderController.class.php#L344): Remove this method "getFooterMessages" to simply inherit it. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)
1. ![MINOR][MINOR] [FounderController.class.php#L353](https://github.com/Wikia/app/blob/d7fbdf873a830ec77a2f879f9359b7ed381c8761/extensions/wikia/Email/Controller/FounderController.class.php#L353): Remove this method "getEmailSpecificFormFields" to simply inherit it. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: extensions/wikia/Email/Controller/FounderController.class.php [hunk 1]
```diff
@@ -610,9 +610,10 @@ protected function getDetailsList() {
 		$discussionsEnabled =
 			\WikiFactory::getVarValueByName( 'wgEnableDiscussions', $this->wikiId );
 		if ( $discussionsEnabled ) {
+			$safeWikiUrl = rtrim( $this->wikiUrl, "/" );
```
Comment (jsutterfield) [inline extensions/wikia/Email/Controller/FounderController.class.php]: `safeWikiUrl` is a bit ambiguous. What's "safe" about it? Maybe `wikiUrlWithoutTrailingSlash` (or something more explicit like that but less wordy).

**Review by jsutterfield — APPROVED** [review]
Small suggestion. Apart from that LGTM.

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [FounderController.class.php#L106](https://github.com/Wikia/app/blob/62f10fb0abefb47ff4cbef387b61a4c2fb6c251f/extensions/wikia/Email/Controller/FounderController.class.php#L106): Immediately return this expression instead of assigning it to the temporary variable "$snippet". [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)
1. ![MINOR][MINOR] [FounderController.class.php#L344](https://github.com/Wikia/app/blob/62f10fb0abefb47ff4cbef387b61a4c2fb6c251f/extensions/wikia/Email/Controller/FounderController.class.php#L344): Remove this method "getFooterMessages" to simply inherit it. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)
1. ![MINOR][MINOR] [FounderController.class.php#L353](https://github.com/Wikia/app/blob/62f10fb0abefb47ff4cbef387b61a4c2fb6c251f/extensions/wikia/Email/Controller/FounderController.class.php#L353): Remove this method "getEmailSpecificFormFields" to simply inherit it. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1185)

[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**